tests/js/spec/views/performance/transactionSpans/spanDetails.spec.tsx

```diff
@@ -552,6 +552,32 @@ describe('Performance > Transaction Spans > Span Summary', function () {
         const nodes = await screen.findAllByText('Self Time Distribution');
         expect(nodes[0]).toBeInTheDocument();
       });
+
+      it('sends min and max to span example query', async function () {
+        const mock = MockApiClient.addMockResponse({
+          url: '/organizations/org-slug/events-spans/',
+          body: {},
+        });
+        const data = initializeData({
+          features: FEATURES,
+          query: {project: '1', transaction: 'transaction', min: '10', max: '120'},
+        });
+
+        render(<SpanDetails params={{spanSlug: 'op:aaaaaaaa'}} {...data} />, {
+          context: data.routerContext,
+          organization: data.organization,
+        });
+
+        expect(mock).toHaveBeenLastCalledWith(
+          '/organizations/org-slug/events-spans/',
+          expect.objectContaining({
+            query: expect.objectContaining({
+              min_exclusive_time: '10',
+              max_exclusive_time: '120',
+            }),
```

Comment:
It's hard to make assertions on the histogram without testing charting library's internal logic since it handles all the rendering logic. It seems to be the way we approached testing charts everywhere in the codebase. I did test that the chart is rendered with good data, empty data and in the error state :)

Reply:
That's fair, I was thinking more along the lines of testing that it renders the correct events in the table, within the min/max time range tho

Reply:
Like for example, you have events A, B, C, D, and then when you apply these query params, the table would only render events B, and C, since they are in that time range. Just throwing a test idea out there :)

Reply:
ah my bad I misunderstood. yeah totally, I can definitely add those test cases 👍

Reply:
oh so since the data is fetched on the container level I'll need to add more table tests in `spanDetails.spec.tsx`, but we already have `suspectSpansTable.spec.tsx`. I'm not sure it'd be nice to add more table tests in span details spec page. I was thinking of testing this integration in acceptance tests if that's cool with you. It seems like a complex test so maybe it'd be easier to orchestrate everything in the acceptance tests?

Reply:
An acceptance test sounds like a good idea to me 👍